src/main/scala/typechecker/TypeCheck.scala

```diff
@@ -470,6 +471,7 @@ object TypeChecker {
       // have static types.
       val rTyp = typ.map(env.resolveType(_) match {
         case TStaticInt(v) => TSizedInt(bitsNeeded(v), false)
+        case _:TRational => TDouble()
```

Comment:
Here (and the next addition below in this diff) is where we're providing the default for rational literals. It is probably a good idea to extend the comment above to describe what we're doing, i.e., providing a default when we can't otherwise possibly know what the right numerical type is.

Reply:
Done